subworkflows/local/profiling.nf

```diff
@@ -129,7 +131,37 @@ workflow PROFILING {
         ch_multiqc_files       = ch_multiqc_files.mix( KRAKEN2_KRAKEN2.out.report )
         ch_versions            = ch_versions.mix( KRAKEN2_KRAKEN2.out.versions.first() )
         ch_raw_classifications = ch_raw_classifications.mix( KRAKEN2_KRAKEN2.out.classified_reads_assignment )
-        ch_raw_profiles        = ch_raw_profiles.mix( KRAKEN2_KRAKEN2.out.report )
+        ch_raw_profiles        = ch_raw_profiles.mix(
+            KRAKEN2_KRAKEN2.out.report
+                // Set the tool to be strictly 'kraken2' instead of potentially 'bracken' for downstream use.
```

Comment:
```suggestion
                // Set the tool to be strictly 'kraken2' instead of potentially 'bracken' for downstream use.
                // Will remain distinct from 'pure' Kraken2 results due to distinct database names in file names.
```

For myself ;)